src/main/scala/chisel3/tester/package.scala

```diff
@@ -164,4 +164,13 @@ package object tester {
   }
 
   implicit def decoupledToDriver[T <: Data](x: DecoupledIO[T]) = new DecoupledDriver(x)
+
+  /**
+    * Simple file name sanitizer
+    * @param name file name to be sanitized
+    * @return
+    */
+  def sanitizeFileName(name: String): String = {
```

Comment:
Should this be part of the testers2 API? I think not, this seems like an internal function.

Reply:
Good question. I have added `//TODO` and API `@note` to build a safe useful api for user access to the directory being used if that is deemed merited. But I'd rather not solve that right now.

Reply:
Can you move this into either internal or experimental? I strongly think this doesn't belong in the tester package.